veracruz-client/src/veracruz_client.rs

```diff
@@ -158,42 +150,46 @@ impl VeracruzClient {
         let client_priv_key = Self::read_private_key(&client_key_filename)?;
 
         // check if the certificate is valid
-        let key_pair = ring::signature::RsaKeyPair::from_der(client_priv_key.0.as_slice())
-            .map_err(|err| VeracruzClientError::RingError(format!("from_der failed:{:?}", err)))?;
-        Self::check_certificate_validity(&client_cert_filename, key_pair.public_key().as_ref())?;
+        let public_key = der::extract_public(client_priv_key.0.as_slice())
+            .map_err(|_| VeracruzClientError::RingError(format!("extract_public failed")))?;
+        Self::check_certificate_validity(&client_cert_filename, public_key.as_ref())?;
 
         let enclave_name = "ComputeEnclave.dev";
 
         let policy_ciphersuite_string = policy.ciphersuite().as_str();
 
         let proxy_service_cert = {
             let certs_pem = policy.proxy_service_cert();
-            let certs =
-                rustls_pemfile::certs(&mut certs_pem.as_bytes()).map_err(|_| {
-                    VeracruzClientError::X509ParserError(format!(
-                        "rustls_pemfile::certs found not certificates"
-                    ))
-                })?;
+            let certs = rustls_pemfile::certs(&mut certs_pem.as_bytes()).map_err(|_| {
+                VeracruzClientError::X509ParserError(format!(
+                    "rustls_pemfile::certs found not certificates"
```

Comment:
sp: "found not" --> "found no"

(It's not your error, but you're touching the line here, so it may as well get fixed.)